### ISSUE TITLE
Ionic2 RC0 Bundle Error

### ISSUE THREAD

**Issue by Chris1308** [driftyco/ionic#8334]
Hey,

i always get the same when webpack recognizes some changes and tries to bundle my project again:

`Error: ENOENT: no such file or directory, mkdir 'C:\Users\user\project\www\build\C:'
    at Error (native)
    at Object.fs.mkdirSync (fs.js:922:18)
    at mkdirpath (C:\Users\user\project\node_modules\rollup\dist\rollup.js:97:6)`

I had to create a custom rollup config to make firebase work, it is here:

`var ngTemplate = require('@ionic/app-scripts/dist/plugins/ng-template').ngTemplate;
var nodeResolve = require('rollup-plugin-node-resolve');
var commonjs = require('rollup-plugin-commonjs');

// https://github.com/rollup/rollup/wiki/JavaScript-API

module.exports = {
  /**
   * entry: The bundle's starting point. This file will
   * be included, along with the minimum necessary code
   * from its dependencies
   */
  entry: './.tmp/app/main.dev.js',

  /**
   * sourceMap: If true, a separate sourcemap file will
   * be created.
   */
  sourceMap: true,

  /**
   * format: The format of the generated bundle
   */
  format: 'iife',

  /**
   * dest: the output filename for the bundle in the buildDir
   */
  dest: 'main.js',

  /**
   * plugins: Array of plugin objects, or a single plugin object.
   * See https://github.com/rollup/rollup/wiki/Plugins for more info.
   */
  plugins: [
    ngTemplate(),
    commonjs({
      include: [
        'node_modules/rxjs/**',
        'node_modules/angularfire2/**',
        'node_modules/firebase/**'
      ],
      namedExports: {
        'node_modules/angularfire2/node_modules/firebase/firebase.js': ['initializeApp', 'auth', 'database'],
        'node_modules/angularfire2/node_modules/firebase/firebase-browser.js': ['initializeApp', 'auth', 'database']
      }
    }),
    nodeResolve({
      module: true,
      jsnext: true,
      main: true,
      browser: true,
      extensions: ['.js']
    })
  ],

  useStrict: false

};`

The only changes here are the useStrict option and the part inside of commonjs().

Any idea what the problem could be? 
(and what can i do to make the second part look as code here?)

**Comment by Chris1308** [comment]
Okay, i found out that the error only appears when i'm working with a custom rollup.config, configured in package.json: 
"config": {
    "ionic_rollup": "config/rollup.config.js"
  }

When i'm making the changes inside @ionic/app-scripts/config/rollup.config.js, everything works fine.

**Comment by jgw96** [comment]
Hello, thanks for opening an issue with us! Since this seems to be an issue with where you were putting your custom rollup.config i am going to be closing this issue for now.

**Comment by smason** [comment]
@Chris1308 was getting the same, unable to get it to recognise my own config.

I searched the source for the error, via:
```shell
find node_modules -type f -exec grep 'not found. Using defaults instead.' '{}' +
```
which pointed to `node_modules/@ionic/app-scripts/dist/util.js`.  From here, I found that it wanted a string that could be passed to `require` instead of a path that we had assumed it wanted.  This still didn't work so debugging the `require`, in `node`, pointed to the default config using a relative path on its `require`s of `ng-template`.  I therefore changed this to:
```javascript
var ngTemplate = require('../dist/plugins/ng-template').ngTemplate;
```
to:
```javascript
var ngTemplate = require('@ionic/app-scripts/dist/plugins/ng-template').ngTemplate;
```
and everything started working for me!
